Resources/Prototypes/Reagents/Consumable/Food/ingredients.yml

```diff
@@ -155,12 +155,14 @@
   meltingPoint: 146
   boilingPoint: 410 # Really high boiling point compared to its melting
   metabolisms:
+    Food:
+      effects:
+        - !type:SatiateHunger # spicy foods satiate hunger better
+          factor: 1.5
     Poison:
       effects:
-        - !type:HealthChange
-          damage:
-            groups:
-              Caustic: 0.5
+        - !type:AdjustTemperature
+          amount: 2500 # thermal energy, not temp
```

Comment:
This is only "balanced" because of human temperature regulation being incredibly OP.